prediction_market_agent/agents/microchain_agent/utils.py

```diff
@@ -111,3 +140,55 @@ def saved_str_to_tmpfile(s: str) -> t.Iterator[str]:
 
     # Finally remove the temporary file
     os.remove(tmp.name)
+
+
+class MechType(str, Enum):
+    PREDICTION_WITH_RESEARCH_REPORT = "prediction-with-research-report-conservative"
+    PREDICTION_ONLINE = "prediction-online"
+
+
+def mech_request(question: str, mech_type: MechType) -> MechResult:
+    private_key = MicrochainAPIKeys().bet_from_private_key.get_secret_value()
+    with saved_str_to_tmpfile(private_key) as tmpfile_path:
+        # Increase gas price to reduce chance of 'out of gas' transaction failures
+        os.environ["MECHX_LEDGER_DEFAULT_GAS_PRICE_STRATEGY"] = "gas_station"
+        response = interact(
+            prompt=question,
+            # Taken from https://github.com/valory-xyz/mech?tab=readme-ov-file#examples-of-deployed-mechs
+            agent_id=6,
```

Comment:
change from `3` -> `6`. This is the same mech that is used by Olas' trader agents (https://aimechs.autonolas.network/mech/0x77af31De935740567Cf4fF1986D04B2c964A786a)

Reply:
Could you please enlighten me about the difference between agent_id and mech_type?